styleguide/examples/sample_pages/form-sample.html

```diff
@@ -6,7 +6,7 @@
           >Skip to navigation</a
         >
         <a class="cads-skip-links__link" href="#cads-main-content"
-          >Skip to content</a
+          >Skip to main content</a
```

Comment:
This is just picking up the wording change from https://github.com/citizensadvice/design-system/pull/2161